util/taints/taints.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Can we move this to internal/util?

I would really prefer if we could "default" to introducing internal utils if we don't explicilty plan to provide them to external consumers of our API.

We have to follow deprecation policies for everything that we expose like that.